core-bundle/tests/DependencyInjection/ContaoCoreExtensionTest.php

```diff
@@ -3797,32 +3805,57 @@ public function testRegistersThePredefinedImageSizes(): void
 
         $this->assertSame('setPredefinedSizes', $methodCalls[0][0]);
 
-        $this->assertSame(
-            [[
-                '_foobar' => [
-                    'width' => 100,
-                    'height' => 200,
+        $expectedSizes = [[
+            '_foo' => [
+                'width' => 150,
+                'height' => 250,
+                'items' => [],
+                'formats' => [],
+            ],
+            '_bar' => [
+                'width' => 150,
+                'items' => [],
+                'formats' => [],
+            ],
+            '_foobar' => [
+                'width' => 100,
+                'height' => 200,
+                'resizeMode' => 'box',
+                'zoom' => 100,
+                'cssClass' => 'foobar-image',
+                'lazyLoading' => true,
+                'densities' => '1x, 2x',
+                'sizes' => '100vw',
+                'skipIfDimensionsMatch' => false,
+                'items' => [[
+                    'width' => 50,
+                    'height' => 50,
                     'resizeMode' => 'box',
                     'zoom' => 100,
-                    'cssClass' => 'foobar-image',
-                    'lazyLoading' => true,
-                    'densities' => '1x, 2x',
-                    'sizes' => '100vw',
-                    'skipIfDimensionsMatch' => false,
-                    'items' => [[
-                        'width' => 50,
-                        'height' => 50,
-                        'resizeMode' => 'box',
-                        'zoom' => 100,
-                        'densities' => '0.5x, 2x',
-                        'sizes' => '50vw',
-                        'media' => '(max-width: 900px)',
-                    ]],
-                    'formats' => [],
-                ],
-            ]],
-            $methodCalls[0][1]
-        );
+                    'densities' => '0.5x, 2x',
+                    'sizes' => '50vw',
+                    'media' => '(max-width: 900px)',
+                ]],
+                'formats' => [],
+            ],
+        ]];
+
+        $sizes = $methodCalls[0][1];
+
+        $sortByKeyRecursive = static function (array &$array) use (&$sortByKeyRecursive) {
```

Comment:
```suggestion
        $sortByKeyRecursive = static function (array &$array) use (&$sortByKeyRecursive): void {
```

Reply:
fixed in e5c26642bf083833261cb9b230cc52d106cc6125